File: CSharp/2189.cs
using System;
using System.Linq;

class URI
{
    static void Main(string[] args)
    {
        int n;
        int cont=1;
        
        while(true)
        {
            n=int.Parse(Console.ReadLine());
            if(n == 0)
            {break;}
            int[] listao= Console.ReadLine().Split().Select(int.Parse).ToArray();

            for(int i =0; i < n; i++)
            {
                if(i+1==listao[i])
                {
                    Console.WriteLine($"Teste {cont}");
                    Console.WriteLine(i+1);
                    Console.WriteLine();
                    cont++;
                    break;
                }
            }

        }
    }
}

File: Python/2189.py
cont=1
while True:
    n=int(input())
    if n==0:
        break
    
    lista =list(map(int,input().split()))
    for i in range(n):
        if(i+1==lista[i]):
            print(f'Teste {cont}')
            cont+=1
            print(i+1)
            print()
            break
            